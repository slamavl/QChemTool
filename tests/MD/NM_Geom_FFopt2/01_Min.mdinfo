

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1626       2.0216E+03     9.6363E-05     4.2114E-04     F29       307

 BOND    =      349.1636  ANGLE   =      315.8065  DIHED      =      682.4507
 VDWAALS =      466.4809  EEL     =       15.5015  HBOND      =        0.0000
 1-4 VDW =      330.9903  1-4 EEL =     -138.7700  RESTRAINT  =        0.0000
