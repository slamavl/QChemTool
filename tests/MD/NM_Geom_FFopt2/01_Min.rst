FGR 
  330
 -10.3670592  -5.1101406   0.2794920 -10.4699702  -5.1619897   1.6246424
 -11.5515611  -5.1486854  -0.3681421  -9.6090835  -6.4229867  -0.2798444
 -10.2347413  -7.4295287   0.3677557  -9.7054042  -6.4861631  -1.6249985
  -9.5383339  -3.8453249  -0.2165838  -9.6358913  -3.8539888  -1.6768778
 -10.3422741  -2.5644369   0.2641007 -10.4567465  -2.5566675   1.6074082
 -11.5210747  -2.5811762  -0.3934371  -9.5366619  -1.2815002  -0.2131265
  -9.6388315  -1.2676340  -1.6730554 -10.3415214  -0.0013704   0.2781418
 -10.4481531   0.0154869   1.6219615 -11.5248016  -0.0167579  -0.3710847
  -9.5372295   1.2801678  -0.2031112  -9.6441542   1.2951826  -1.6625312
 -10.3373160   2.5653230   0.2902301 -10.4360150   2.5943454   1.6346272
 -11.5252480   2.5498138  -0.3510091  -9.5392851   3.8455819  -0.2022139
  -9.6768041   3.8462114  -1.6587460 -10.3405169   5.1190108   0.3386461
 -10.3522810   5.2166670   1.6842031 -11.5751784   5.1243843  -0.2053968
  -9.6034044   6.3957214  -0.3385714  -9.6938691   6.3570877  -1.6841278
 -10.2253765   7.4622854   0.2054806  -8.0999631   6.3385350   0.2022775
  -7.3902349   7.6697755  -0.2901677  -7.4647274   7.7407447  -1.6345641
  -7.9707571   8.7063120   0.3510787  -8.1692567   6.4573112   1.6588104
  -7.3535815   5.0853912  -0.2414346  -7.3859884   5.1044369  -1.7014348
  -8.0808372   3.8257551   0.2414889  -8.1135254   3.8442923   1.7014893
  -7.3578025   2.5494412  -0.2240544  -7.3896084   2.5532744  -1.6837009
  -8.0825903   1.2759605   0.2483581  -8.1265405   1.2798664   1.7080274
  -7.3581354  -0.0007370  -0.2213742  -7.3857414   0.0016408  -1.6810040
  -8.0839695  -1.2783763   0.2443978  -8.1326115  -1.2808294   1.7039053
  -7.3590962  -2.5522726  -0.2264068  -7.3863049  -2.5544491  -1.6861673
  -8.0858110  -3.8290504   0.2370651  -8.1366248  -3.8420014   1.6966179
  -7.3589938  -5.0879313  -0.2373337  -7.3955773  -5.1254387  -1.6968881
  -8.0993716  -6.3377169   0.2162754  -8.1556949  -6.4178939   1.6765667
  -7.3920572  -7.6743920  -0.2644090  -7.4425283  -7.7773923  -1.6077195
  -7.9959803  -8.6868986   0.3930969  -5.8782082  -7.6181984   0.2128614
  -5.9173256  -7.7136332   1.6727879  -5.1720100  -8.9552908  -0.2784054
  -5.2106912  -9.0560466  -1.6222276  -5.7770016  -9.9723519   0.3707906
  -3.6600321  -8.8995409   0.2028899  -3.7005315  -8.9996668   1.6623075
  -2.9470955 -10.2350131  -0.2904490  -2.9712762 -10.3349830  -1.6348480
  -3.5545177 -11.2560412   0.3507613  -1.4393559 -10.1840454   0.2020356
  -1.5076095 -10.3034728   1.6585644  -0.7371464 -11.5146460  -0.3388221
  -0.6584220 -11.5736469  -1.6843777  -1.3498457 -12.5865826   0.2051927
   0.7370564 -11.5146626   0.3384330   0.6583315 -11.5737057   1.6839868
   1.3497474 -12.5865866  -0.2056159   1.4392761 -10.1840501  -0.2023824
   2.9470158 -10.2350456   0.2900990   2.9711969 -10.3350597   1.6344947
   3.5544294 -11.2560574  -0.3511453   1.5075276 -10.3034305  -1.6589151
   0.7272114  -8.9110927   0.2413279   0.7274646  -8.9486995   1.7013275
  -0.7272810  -8.9110791  -0.2416327  -0.7275345  -8.9486378  -1.7016336
  -1.4710861  -7.6467560   0.2239077  -1.4837079  -7.6762367   1.6835535
  -2.9363342  -7.6376825  -0.2485433  -2.9548885  -7.6776774  -1.7082136
  -3.6797617  -6.3719380   0.2211865  -3.6915449  -6.3970547   1.6808156
  -5.1491343  -6.3616910  -0.2446250  -5.1755408  -6.4025682  -1.7041338
  -5.8899271  -5.0969852   0.2261781  -5.9054560  -5.1194819   1.6859377
  -5.1543443  -3.8244126  -0.2333567  -5.1647009  -3.8309462  -1.6930510
  -5.8892367  -2.5515443   0.2331667  -5.9001118  -2.5572687   1.6928606
  -5.1545389  -1.2759534  -0.2296733  -5.1630022  -1.2761780  -1.6892642
  -5.8880103  -0.0008567   0.2359915  -5.8970359  -0.0013846   1.6956681
  -5.1536052   1.2743603  -0.2296623  -5.1624972   1.2750567  -1.6892582
  -5.8869102   2.5499327   0.2334377  -5.8939578   2.5551578   1.6931342
  -5.1517330   3.8232885  -0.2334009  -5.1597900   3.8267841  -1.6930973
  -5.8867409   5.0973687   0.2240993  -5.9059541   5.1229930   1.6837460
  -5.1462612   6.3617892  -0.2483145  -5.1716274   6.3979022  -1.7079835
  -5.8772115   7.6194447   0.2031625  -5.9436666   7.7045342   1.6625832
  -5.1695057   8.9567464  -0.2780931  -5.2374283   9.0406664  -1.6219122
  -5.7478076   9.9891942   0.3711389  -3.6584483   8.8997700   0.2131649
  -2.9501910  10.2389138  -0.2640657  -3.0141626  10.3341665  -1.6073729
  -3.5250829  11.2681587   0.3934760  -3.7215317   8.9813171   1.6730943
  -2.9348200   7.6401345  -0.2443654  -2.9570243   7.6834887  -1.7038727
  -3.6783783   6.3727290   0.2214088  -3.6942316   6.3954449   1.6810387
  -2.9432247   5.0996184  -0.2359638  -2.9472879   5.1077023  -1.6956403
  -3.6803998   3.8260010   0.2296908  -3.6854408   3.8333499   1.6892868
  -2.9448981   2.5504375  -0.2320867  -2.9467591   2.5513146  -1.6916438
  -3.6811816   1.2751659   0.2321080  -3.6828644   1.2763348   1.6916650
  -2.9457510  -0.0003781  -0.2313347  -2.9481832  -0.0002679  -1.6908780
  -3.6820326  -1.2759566   0.2321258  -3.6842305  -1.2777631   1.6916827
  -2.9460411  -2.5507290  -0.2322390  -2.9486668  -2.5517075  -1.6917960
  -3.6823135  -3.8259595   0.2295221  -3.6867781  -3.8331979   1.6891128
  -2.9447807  -5.0987118  -0.2361417  -2.9497127  -5.1062432  -1.6958185
  -1.4732201  -5.1003249   0.2295504  -1.4771011  -5.1083943   1.6891462
  -0.7351925  -6.3731711  -0.2335481  -0.7341540  -6.3818672  -1.6932447
   0.7351427  -6.3731848   0.2333279   0.7341042  -6.3819302   1.6930242
   1.4710261  -7.6467600  -0.2241710   1.4836466  -7.6761919  -1.6838178
   2.9362746  -7.6377138   0.2482789   2.9548299  -7.6777581   1.7079479
   3.6599621  -8.8995628  -0.2031973   3.7004584  -8.9996404  -1.6626184
   5.1719404  -8.9553406   0.2780930   5.2106232  -9.0561421   1.6219117
   5.7769230  -9.9723845  -0.3711385   5.8781481  -7.6182371  -0.2131303
   7.3919979  -7.6744590   0.2641342   7.4424713  -7.7775068   1.6074410
   7.9959115  -8.6869473  -0.3934087   5.9172615  -7.7136222  -1.6730602
   5.1490849  -6.3617398   0.2444007   5.1754931  -6.4026678   1.7039081
   3.6797115  -6.3719592  -0.2214089   3.6914926  -6.3970262  -1.6810389
   2.9447410  -5.0987429   0.2359637   2.9496738  -5.1063248   1.6956403
   1.4731800  -5.1003285  -0.2297275   1.4770602  -5.1083480  -1.6893236
   0.7362588  -3.8255844   0.2320496   0.7360507  -3.8276557   1.6916066
  -0.7362887  -3.8255707  -0.2321827  -0.7360806  -3.8275915  -1.6917398
  -1.4732289  -2.5508980   0.2312595  -1.4743872  -2.5530807   1.6908028
  -0.7364963  -1.2754928  -0.2317145  -0.7368115  -1.2757909  -1.6912468
  -1.4728632  -0.0000700   0.2316769  -1.4733162  -0.0002155   1.6912092
  -0.7363621   1.2755783  -0.2316699  -0.7364695   1.2760473  -1.6912023
  -1.4725270   2.5512952   0.2313486  -1.4738312   2.5533427   1.6908919
  -0.7359767   3.8267184  -0.2321124  -0.7355180   3.8295285  -1.6916693
  -1.4722203   5.1019494   0.2296933  -1.4762498   5.1093881   1.6892841
  -0.7348674   6.3760069  -0.2331480  -0.7353542   6.3882897  -1.6928420
  -1.4691541   7.6493118   0.2264311  -1.4808655   7.6739616   1.6861916
  -0.7267820   8.9170488  -0.2370433  -0.7409794   8.9675318  -1.6965960
  -1.4389370  10.1831131   0.2166099  -1.4802039  10.2719320   1.6769042
  -0.7579292  11.5332127  -0.2794697  -0.7644875  11.6482604  -1.6246201
  -1.3167882  12.5782984   0.3681666   0.7580193  11.5331979   0.2798597
   0.7645784  11.6482027   1.6250137   1.3168865  12.5782999  -0.3677434
   1.4390168  10.1831089  -0.2162627   2.9502708  10.2388821   0.2644165
   3.0142417  10.3340901   1.6077268   3.5251714  11.2681441  -0.3930909
   1.4802857  10.2719750  -1.6765541   0.7268517   8.9170353   0.2373487
   0.7410495   8.9674699   1.6969031   1.4692141   7.6493079  -0.2261675
   1.4809268   7.6740065  -1.6859272   0.7349172   6.3759933   0.2333685
   0.7354041   6.3882265   1.6930628   1.4722603   5.1019458  -0.2295160
   1.4762907   5.1094345  -1.6891066   0.7360066   3.8267046   0.2322455
   0.7355479   3.8294642   1.6918025   1.4725470   2.5512917  -0.2312595
   1.4738516   2.5533900  -1.6908027   0.7363721   1.2755644   0.2317145
   0.7364794   1.2759825   1.6912468   1.4728632  -0.0000734  -0.2316769
   1.4733162  -0.0001679  -1.6912093   0.7364863  -1.2755066   0.2316699
   0.7368016  -1.2758556   1.6912023   1.4732088  -2.5509015  -0.2313486
   1.4743668  -2.5530335  -1.6908919   2.9460212  -2.5507602   0.2321494
   2.9486474  -2.5517898   1.6917064   3.6822834  -3.8259803  -0.2296567
   3.6867467  -3.8331680  -1.6892475   5.1543147  -3.8244611   0.2332209
   5.1646724  -3.8310465   1.6929150   5.8898868  -5.0970234  -0.2263596
   5.9054132  -5.1194692  -1.6861201   7.3589544  -5.0879973   0.2371499
   7.3955401  -5.1255573   1.6967029   8.0993215  -6.3377726  -0.2165053
   8.1556405  -6.4178984  -1.6767995   9.6090341  -6.4230720   0.2796071
   9.7053580  -6.4862981   1.6247587  10.2346822  -7.4295952  -0.3680314
  10.3670185  -5.1102114  -0.2796835  10.4699254  -5.1620122  -1.6248360
  11.5515219  -5.1487890   0.3679459   9.5383042  -3.8454075   0.2164408
  10.3422537  -2.5645080  -0.2641973  10.4567244  -2.5566900  -1.6075047
  11.5210550  -2.5812807   0.3933383   9.6358639  -3.8541259   1.6767344
   8.0857807  -3.8291051  -0.2372050   8.1365920  -3.8420042  -1.6967583
   7.3590765  -2.5523384   0.2263141   7.3862864  -2.5545680   1.6860744
   5.8892166  -2.5515821  -0.2332581   5.9000903  -2.5572549  -1.6929522
   5.1545291  -1.2760019   0.2296278   5.1629928  -1.2762786   1.6892186
   3.6820226  -1.2759772  -0.2321708   3.6842200  -1.2777325  -1.6917279
   2.9457510  -0.0004093   0.2313347   2.9481833  -0.0003506   1.6908779
   3.6811917   1.2751453  -0.2320630   3.6828749   1.2763655  -1.6916201
   2.9449180   2.5504064   0.2321762   2.9467785   2.5512322   1.6917333
   3.6804299   3.8259802  -0.2295564   3.6854722   3.8333798  -1.6891521
   2.9432644   5.0995872   0.2361418   2.9473268   5.1076206   1.6958187
   3.6784284   6.3727078  -0.2211862   3.6942838   6.3954734  -1.6808154
   2.9348796   7.6401033   0.2446300   2.9570829   7.6834079   1.7041388
   3.6585182   8.8997485  -0.2128571   3.7216047   8.9813438  -1.6727836
   5.1695751   8.9566965   0.2784060   5.2374961   9.0405703   1.6222281
   5.7478863   9.9891618  -0.3707898   5.8772716   7.6194057  -0.2028934
   7.3902942   7.6697074   0.2904425   7.4647841   7.7406287   1.6348416
   7.9708259   8.7062620  -0.3507660   5.9437304   7.7045448  -1.6623110
   5.1463105   6.3617403   0.2485388   5.1716751   6.3978023   1.7082092
   5.8867811   5.0973305  -0.2239178   5.9059968   5.1230057  -1.6835636
   5.1517627   3.8232399   0.2335366   5.1598184   3.8266837   1.6932332
   5.8869303   2.5498949  -0.2333465   5.8939793   2.5551718  -1.6930428
   5.1536151   1.2743118   0.2297076   5.1625067   1.2749560   1.6893036
   5.8880103  -0.0008943  -0.2359916   5.8970359  -0.0013703  -1.6956682
   7.3581354  -0.0008026   0.2213741   7.3857415   0.0015217   1.6810039
   8.0839594  -1.2784307  -0.2444448   8.1326005  -1.2808313  -1.7039524
   9.5366521  -1.2815828   0.2130786   9.6388225  -1.2677715   1.6730079
  10.3415214  -0.0014410  -0.2781419  10.4481533   0.0154652  -1.6219610
  11.5248015  -0.0168617   0.3710840   9.5372394   1.2800856   0.2031588
   9.6441635   1.2950455   1.6625795   8.0826005   1.2759063  -0.2483114
   8.1265515   1.2798647  -1.7079806   7.3578223   2.5493755   0.2241468
   7.3896270   2.5531555   1.6837935   8.0808675   3.8257006  -0.2413494
   8.1135583   3.8442899  -1.7013491   7.3536208   5.0853250   0.2416182
   7.3860254   5.1043181   1.7016191   8.1000132   6.3384788  -0.2020477
   8.1693111   6.4573060  -1.6585762   9.6034535   6.3956340   0.3388072
   9.6939145   6.3569506   1.6843625  10.2254352   7.4622130  -0.2052042
  10.3405578   5.1189424  -0.3384549  10.3523261   5.2166474  -1.6840083
  11.5752179   5.1242865   0.2055914   9.5393149   3.8454998   0.2023566
   9.6768316   3.8460747   1.6588889  10.3373365   2.5652528  -0.2901337
  11.5252675   2.5497108   0.3511066  10.4360376   2.5943239  -1.6345296
