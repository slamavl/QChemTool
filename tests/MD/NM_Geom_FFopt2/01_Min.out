
          -------------------------------------------------------
          Amber 14 SANDER                              2014
          -------------------------------------------------------

| Run on 03/18/2018 at 11:56:49

|   Executable path: sander
| Working directory: /mnt/sda2/Dropbox/PhD/Programy/Python/QChemTool/tests/MD/NM_Geom_FFopt2
|          Hostname: Unknown
  [-O]verwriting output

File Assignments:
|  MDIN: 01_Min.in                                                             
| MDOUT: 01_Min.out                                                            
|INPCRD: nab_input.inpcrd                                                      
|  PARM: nab_input.prmtop                                                      
|RESTRT: 01_Min.rst                                                            
|  REFC: refc                                                                  
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: mden                                                                  
| MDCRD: 01_Min.mdcrd                                                          
|MDINFO: 01_Min.mdinfo                                                         
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

minimize                                                                       
 &cntrl                                                                        
  imin=1,                                                                      
  ntb=0,                                                                       
  ntx=1,                                                                       
  irest=0,                                                                     
  maxcyc=10000,                                                                
  ncyc=4000,                                                                   
  ntpr=100,                                                                    
  ntwx=10,                                                                     
  cut=999.0,                                                                   
 /                                                                             

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
|    NONPERIODIC   ntb=0 and igb=0: Setting up nonperiodic simulation
|  *** cutoff > system size, list only builds once
|Largest sphere to fit in unit cell has radius =    31.728
| New format PARM file being parsed.
| Version =    1.000 Date = 03/18/18 Time = 11:56:48
 NATOM  =     330 NTYPES =       2 NBONH =       0 MBONA  =     390
 NTHETH =       0 MTHETA =     900 NPHIH =       0 MPHIA  =    3126
 NHPARM =       0 NPARM  =       0 NNB   =    2998 NRES   =       1
 NBONA  =     390 NTHETA =     900 NPHIA =    3126 NUMBND =       5
 NUMANG =      10 NPTRA  =       8 NATYP =       4 NPHB   =       0
 IFBOX  =       0 NMXRS  =     330 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0


|     Memory Use     Allocated
|     Real               30567
|     Hollerith            993
|     Integer           174685
|     Max Pairs          54285
|     nblistReal          3960
|     nblist Int       1528485
|       Total             7138 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

FGR                                                                             

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     100, ntrx    =       1, ntwr    =     500
     iwrap   =       0, ntwx    =      10, ntwv    =       0, ntwe    =       0
     ioutfm  =       0, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       0, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       0

Energy minimization:
     maxcyc  =   10000, ncyc    =    4000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010
|  INFO: Old style inpcrd file read


--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

FGR                                                                             
 begin time read from input coords =     0.000 ps

 Number of triangulated 3-point waters found:        0

     Sum of charges from parm topology file =  -0.00000002
     Forcing neutrality...

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

 ---------------------------------------------------

     eedmeth=4: Setting switch to one everywhere

 ---------------------------------------------------
| Local SIZE OF NONBOND LIST =      51288
| TOTAL SIZE OF NONBOND LIST =      51288


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1       2.3074E+03     2.4137E+01     8.4969E+01     C1          1

 BOND    =      768.9183  ANGLE   =      458.8334  DIHED      =      686.2505
 VDWAALS =      290.9881  EEL     =       13.3706  HBOND      =        0.0000
 1-4 VDW =      226.0755  1-4 EEL =     -137.0145  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    100       2.0264E+03     4.6143E-01     1.5338E+00     C149      328

 BOND    =      400.0565  ANGLE   =      334.4667  DIHED      =      681.8323
 VDWAALS =      423.4371  EEL     =       14.8712  HBOND      =        0.0000
 1-4 VDW =      309.9144  1-4 EEL =     -138.1716  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    200       2.0217E+03     7.0584E-02     1.7865E-01     C64       149

 BOND    =      355.9445  ANGLE   =      318.5636  DIHED      =      682.3909
 VDWAALS =      460.0638  EEL     =       15.4061  HBOND      =        0.0000
 1-4 VDW =      328.0322  1-4 EEL =     -138.6761  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    300       2.0216E+03     3.6192E-02     1.6175E-01     C76       175

 BOND    =      350.1187  ANGLE   =      316.3066  DIHED      =      682.4718
 VDWAALS =      465.4068  EEL     =       15.4850  HBOND      =        0.0000
 1-4 VDW =      330.5927  1-4 EEL =     -138.7523  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    400       2.0216E+03     3.8354E-03     1.3352E-02     C69       157

 BOND    =      349.2952  ANGLE   =      315.9227  DIHED      =      682.4764
 VDWAALS =      466.2341  EEL     =       15.4973  HBOND      =        0.0000
 1-4 VDW =      330.9645  1-4 EEL =     -138.7649  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500       2.0216E+03     2.6389E-03     1.1424E-02     F172       16

 BOND    =      349.1651  ANGLE   =      315.8596  DIHED      =      682.4708
 VDWAALS =      466.3840  EEL     =       15.4997  HBOND      =        0.0000
 1-4 VDW =      331.0127  1-4 EEL =     -138.7674  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    600       2.0216E+03     3.3852E-03     1.1606E-02     C138      305

 BOND    =      349.1535  ANGLE   =      315.8360  DIHED      =      682.4658
 VDWAALS =      466.4229  EEL     =       15.5004  HBOND      =        0.0000
 1-4 VDW =      331.0138  1-4 EEL =     -138.7683  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    700       2.0216E+03     2.4196E-03     8.6821E-03     C76       175

 BOND    =      349.1498  ANGLE   =      315.8319  DIHED      =      682.4616
 VDWAALS =      466.4403  EEL     =       15.5007  HBOND      =        0.0000
 1-4 VDW =      331.0082  1-4 EEL =     -138.7688  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    800       2.0216E+03     3.0666E-03     1.3405E-02     C76       175

 BOND    =      349.1569  ANGLE   =      315.8206  DIHED      =      682.4589
 VDWAALS =      466.4515  EEL     =       15.5009  HBOND      =        0.0000
 1-4 VDW =      331.0039  1-4 EEL =     -138.7691  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    900       2.0216E+03     2.5758E-03     1.0253E-02     C119      267

 BOND    =      349.1554  ANGLE   =      315.8204  DIHED      =      682.4564
 VDWAALS =      466.4597  EEL     =       15.5011  HBOND      =        0.0000
 1-4 VDW =      330.9999  1-4 EEL =     -138.7694  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000       2.0216E+03     2.1474E-03     8.7161E-03     C119      267

 BOND    =      349.1607  ANGLE   =      315.8131  DIHED      =      682.4549
 VDWAALS =      466.4657  EEL     =       15.5012  HBOND      =        0.0000
 1-4 VDW =      330.9974  1-4 EEL =     -138.7696  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1100       2.0216E+03     5.4032E-04     1.7666E-03     C138      305

 BOND    =      349.1606  ANGLE   =      315.8121  DIHED      =      682.4536
 VDWAALS =      466.4703  EEL     =       15.5013  HBOND      =        0.0000
 1-4 VDW =      330.9952  1-4 EEL =     -138.7697  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1200       2.0216E+03     3.6725E-04     1.5244E-03     F100      159

 BOND    =      349.1611  ANGLE   =      315.8108  DIHED      =      682.4527
 VDWAALS =      466.4736  EEL     =       15.5014  HBOND      =        0.0000
 1-4 VDW =      330.9936  1-4 EEL =     -138.7698  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1300       2.0216E+03     4.0357E-04     1.3577E-03     C76       175

 BOND    =      349.1625  ANGLE   =      315.8087  DIHED      =      682.4520
 VDWAALS =      466.4762  EEL     =       15.5014  HBOND      =        0.0000
 1-4 VDW =      330.9925  1-4 EEL =     -138.7699  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1400       2.0216E+03     2.6236E-04     9.2165E-04     F29       307

 BOND    =      349.1625  ANGLE   =      315.8082  DIHED      =      682.4514
 VDWAALS =      466.4781  EEL     =       15.5015  HBOND      =        0.0000
 1-4 VDW =      330.9915  1-4 EEL =     -138.7700  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500       2.0216E+03     3.9672E-04     1.4873E-03     C119      267

 BOND    =      349.1636  ANGLE   =      315.8068  DIHED      =      682.4511
 VDWAALS =      466.4796  EEL     =       15.5015  HBOND      =        0.0000
 1-4 VDW =      330.9909  1-4 EEL =     -138.7700  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1600       2.0216E+03     3.3630E-04     1.1867E-03     C134      299

 BOND    =      349.1632  ANGLE   =      315.8069  DIHED      =      682.4507
 VDWAALS =      466.4806  EEL     =       15.5015  HBOND      =        0.0000
 1-4 VDW =      330.9904  1-4 EEL =     -138.7700  RESTRAINT  =        0.0000


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1626       2.0216E+03     9.6363E-05     4.2114E-04     F29       307

 BOND    =      349.1636  ANGLE   =      315.8065  DIHED      =      682.4507
 VDWAALS =      466.4809  EEL     =       15.5015  HBOND      =        0.0000
 1-4 VDW =      330.9903  1-4 EEL =     -138.7700  RESTRAINT  =        0.0000

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|    Read coords time           0.00 ( 0.01% of Total)
|                Build the list             0.00 (91.01% of List )
|                Other                      0.00 ( 8.99% of List )
|             List time                  0.00 ( 0.05% of Nonbo)
|                   Short_ene time             2.67 (99.80% of Direc)
|                   Other                      0.01 ( 0.20% of Direc)
|                Direct Ewald time          2.67 (96.30% of Ewald)
|                Virial junk                0.00 ( 0.01% of Ewald)
|                Start synchronizatio       0.00 ( 0.01% of Ewald)
|                Other                      0.10 ( 3.68% of Ewald)
|             Ewald time                 2.77 (100.0% of Nonbo)
|             Other                      0.00 ( 0.01% of Nonbo)
|          Nonbond force              2.78 (73.79% of Force)
|          Bond/Angle/Dihedral        0.98 (26.17% of Force)
|          Other                      0.00 ( 0.03% of Force)
|       Force time                 3.76 (100.0% of Runmd)
|    Runmd Time                 3.76 (97.44% of Total)
|    Other                      0.10 ( 2.55% of Total)
| Total time                 3.86 (100.0% of ALL  )

| Number of list builds   :          1

| Highest rstack allocated:       1595
| Highest istack allocated:        319
|           Job began  at 11:56:49.040  on 03/18/2018
|           Setup done at 11:56:49.049  on 03/18/2018
|           Run   done at 11:56:52.901  on 03/18/2018
|     wallclock() was called   37433 times
